src/components/Garden/Stake/StakingMovements.js

```diff
@@ -85,10 +85,13 @@ function StakingMovements({ stakingMovements, token }) {
 
   const [selectedPage, setSelectedPage] = useState(0)
 
-  const handlePageChange = useCallback(page => {
+  const handlePageChange = useCallback((page) => {
     setSelectedPage(page)
   }, [])
 
+  const getProposalType = (disputableAddress) =>
+    disputableAddress === config.voting.id ? 'Vote' : 'Decision'
```

Comment:
```suggestion
    disputableAddress === config.voting.id ? 'Decision' : 'Proposal'
```

Reply:
The `decision` actions are the ones that alter the Gardens DNA (use the Disputable Voting Aragon app). The `proposals` are both of type suggestion and funding (when requesting funds).

Reply:
Sure Gabi 👍🏼